packages/faucet/src/database/migrations/0000_fast_thunderball.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE "requests" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"name" varchar(50) NOT NULL,
+	"company" varchar(150),
+	"email_address" varchar(254) NOT NULL,
+	"from_address" varchar(44) NOT NULL,
+	"to_address" varchar(44) NOT NULL,
+	"hash" varchar(64) NOT NULL,
+	"marketing_optin" boolean DEFAULT false NOT NULL,
+	"amount" bigint NOT NULL,
+	"denom" varchar(20) NOT NULL,
+	"created_at" timestamp DEFAULT now() NOT NULL,
+	"country" varchar(2) DEFAULT 'XX' NOT NULL
```

Comment:
- Would add a constraint e.g. `CHECK (country ~ '^[A-Z]{2}$')`
- Also, since `XX` is not a valid country, would it not be better to allow null in this?

Reply:
I added XX because it's [CF's value for clients without country code](https://developers.cloudflare.com/fundamentals/reference/http-headers/#cf-ipcountry).
Happy to change if necessary.

The reason I didn't add additional safety checks is because we already have the default fallback value for this:

`private getCountryFromRequest(context: Context): string {
    return context.get('CF-IPCountry') || 'XX';
  }`